packages/ai-chat-ui/src/browser/chat-input-widget.tsx

```diff
@@ -584,9 +621,9 @@ const ChangeSetBox: React.FunctionComponent<{ changeSet: ChangeSetUI }> = ({ cha
                     </li>
                 ))}
             </ul>
-        </div>
-    </div>
-);
+        </div >
+    </div >
```

Comment:
These are weird formatting changes. I'm surprised the linter accepts them

Reply:
This appears to be acceptable according to [the spec](https://dev.w3.org/html5/spec-LC/syntax.html#end-tags) for HTML tags, and the rest of the code base has a mix, admittedly with a majority without space. I don't _think_ I've configured anything specific to this case, so this is likely to be the default behavior of the TS formatter on JSX?

Reply:
Overall it's a nitpick as it does not really matter. So feel free to keep it.

It's just strange to me that we change these two `div` elements here specifically (they are not even new ones), and for example not the `ul` or `li` elements. As this is very arbitrary and not even consistent within the same JSX block I would like to suggest removing this formatting change.